src/index.css

```diff
@@ -3,45 +3,72 @@
 }
 
 a {
-  color: black;
+  color: #121212;
   text-decoration: none;
+  font-size: 0.9rem;
+}
+
+.catg {
+  opacity: 0.5;
 }
 
 .head {
   display: flex;
   justify-content: space-between;
+  align-items: center;
   width: 94%;
   margin: 0 auto;
 }
 
 nav {
   display: flex;
   align-items: center;
-  gap: 2rem;
-  color: rgb(0, 145, 255);
+  gap: 1.9rem;
+  color: #0290ff;
 }
 
 .book-parent-div {
   display: flex;
-  align-items: center;
+  align-items: start;
+  max-height: 9.5rem;
   gap: 2%;
-  width: 90%;
-  margin: 5rem auto 0;
+  width: 92%;
+  position: relative;
+  top: 4rem;
   border-top: 1px solid lightgray;
+  border-bottom: 1px solid lightgray;
+  margin: 1rem auto;
 }
 
 .book-first-div {
   width: 48%;
   display: flex;
   flex-direction: column;
   align-items: start;
-  margin-top: 1rem;
 }
 
 .margin-zero {
   margin: 0.3rem 0;
 }
 
+h3 {
+  opacity: 0.5;
+  font-size: 0.9rem;
+}
+
+h2 {
+  font-size: 1.8rem;
+  font-weight: bold;
+}
+
+.list-btn {
+  color: #4386bf;
+  font-size: 0.9rem;
+  border: none;
+  background-color: transparent;
+  cursor: pointer;
+}
+
```

Comment:
- [ ] I suggest adjusting the project style to match the [Zeplin Template](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378). The body background color should be `rgba(0, 0, 0, 0.05)` and the book's title and project title font family should match the template design. Also, ensure the book title has a color of `#121212` to match the template design. The nav background should be `#fff` and the book's card background should be `#fff` with a border-left line and a border-right line.

### Your design
![image](https://github.com/MohamedRoshdy2021/Book-store/assets/92303891/59dda22a-10bf-4af5-aa8d-69686a6d06a9)


### Zeplin template design
![Screenshot from 2023-08-26 16-01-30](https://github.com/MohamedRoshdy2021/Book-store/assets/92303891/0a4d23ad-6643-4222-9f90-4b4d43338405)